bigflow/log.py

```diff
@@ -198,10 +148,13 @@ def init_logging(config: LogConfigDict, workflow_id: str, banner=True):
                Workflow logs (all runs): {workflow_logs_link}
                Only this run logs: {this_execution_logs_link}
                ***********************************************************"""))
-    
-    gcp_logger_handler = GCPLoggerHandler(gcp_project_id, log_name, labels)
+    gcp_logger_handler = create_gcp_log_handler(gcp_project_id, log_name, labels)
     gcp_logger_handler.setLevel(log_level or logging.INFO)
 
+    # Disable logs from 'google.cloud.logging'
```

Comment:
As I understand we set "google.cloud.logging" level to WARNING to avoid sending it through gcp_logger_handler. But what if log_level from line 121 will be also WARNING? I think in line 156 we should setlevel as WARNING +1

Reply:
'log_level' is used only to filter log messages. It is not related to level, at which messages may come from getLogger('google.cloud.logger')

In theory there are still might be cases, when some 'WARN' message might be generated and might cause recursive logging issue. So I've specified 'propagate = False' + attach separate console handler to this logger.